### PR TITLE
Add favicon

### PR DESCRIPTION
**Pull request by MartinaVilimova**
Added favicon in common ico format.

I found out on the web that you can change the favicon according to the theme (light / dark), but we have to have the svg icon. You can write media queries to svg. Unfortunately, it didn't work for me. I do not know why. So far, I have created an icon with black and white and had the required sizes generated.

If he really wants a dark mode favicon, I can try to write a condition for changing the icon. I haven't tried yet.

I tried the favicon as text - unreadable. Then a few pictures and finally I chose the users icon.

image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,15 +2,35 @@
...
 			name="description"
 			content="Web application that connects programmers based on their experience, interests and the type of person they are looking for (mentor, buddy, mentee) through matching and contact sharing."
 		/>
-		<!-- <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" /> -->
-		<link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+		<!-- favicon -->
+		<link
+			rel="apple-touch-icon"
+			sizes="180x180"
+			href="/apple-touch-icon.png"
```
Comment (macajind) [inline public/index.html:15]: I would use the `%PUBLIC_URL%` here. :wink:
https://create-react-app.dev/docs/using-the-public-folder/
```suggestion
			href="%PUBLIC_URL%/apple-touch-icon.png"
```

File: public/index.html [hunk 2]
```diff
@@ -2,15 +2,35 @@
...
+		/>
+		<link
+			rel="icon"
+			type="image/png"
+			sizes="32x32"
+			href="/favicon-32x32.png"
```
Comment (macajind) [inline public/index.html:21]: Also here. :grin:
```suggestion
			href="%PUBLIC_URL%/favicon-32x32.png"
```

File: public/index.html [hunk 3]
```diff
@@ -2,15 +2,35 @@
...
+		/>
+		<link
+			rel="icon"
+			type="image/png"
+			sizes="16x16"
+			href="/favicon-16x16.png"
```
Comment (macajind) [inline public/index.html:27]: Here. :grin:
```suggestion
			href="%PUBLIC_URL%/favicon-16x16.png"
```

File: public/index.html [hunk 4]
```diff
@@ -2,15 +2,35 @@
...
+		/>
+		<link rel="manifest" href="/site.webmanifest" />
```
Comment (macajind) [inline public/index.html:29]: Here. :grin:
```suggestion
		<link rel="manifest" href="%PUBLIC_URL%/site.webmanifest" />
```

File: public/index.html [hunk 5]
```diff
@@ -2,15 +2,35 @@
...
+		<!-- set a theme color -->
+		<meta name="msapplication-config" content="/browserconfig.xml" />
```
Comment (macajind) [inline public/index.html:31]: And here. :grin:
```suggestion
		<meta name="msapplication-config" content="%PUBLIC_URL%/browserconfig.xml" />
```

File: public/index.html [hunk 6]
```diff
@@ -2,15 +2,35 @@
...
 		<meta charset="utf-8" />
-		<link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
 		<meta name="theme-color" content="#264653" />
 		<meta
 			name="description"
 			content="Web application that connects programmers based on their experience, interests and the type of person they are looking for (mentor, buddy, mentee) through matching and contact sharing."
 		/>
-		<!-- <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" /> -->
-		<link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```
Comment (macajind) [inline public/index.html:13]: Why did you remove this default manifest? :thinking:

You technically did a copy in the `site.webmanifest`, so now we have 2 very similar manifests and only linking one. :smile:

File: public/manifest.json [hunk 7]
```diff
@@ -11,5 +11,17 @@
 	"display": "browser",
 	"name": "[: aj tý :] spojka",
 	"short_name": "spojka.it",
-	"theme_color": "#264653"
+	"theme_color": "#264653",
+	"icons": [
+		{
+			"src": "/android-chrome-192x192.png",
```
Comment (macajind) [inline public/manifest.json:17]: No sure about `src`  se to `/` here. :sweat_smile:

**Review by macajind — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 6 above)
Comment (MartinaVilimova) [reply public/index.html:13]: Zkopíroval jsem kód vygenerovaný faviconou. Takže jsem si myslel, že webmanifest je součástí favicony. Měl bych tedy odstranit webmanifest a nechat tam jen manifest.json?

image

Comment (macajind) [reply public/index.html:13]: Yeah, I would leave the default one, because they are almost same. :+1:

Comment (MartinaVilimova) [reply public/index.html:13]: ok, fixed 👍

**Review by agnes97 — APPROVED** [review]

**Review by macajind — APPROVED** [review]
:1st_place_medal:
